sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/internal/SensorContextTester.java

```diff
@@ -336,4 +337,9 @@ public void addContextProperty(String key, String value) {
   public Map<String, String> getContextProperties() {
     return ImmutableMap.copyOf(sensorStorage.contextProperties);
   }
+
+  @Override
+  public void markForPublishing(InputFile inputFile) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1186)